cmd/postgres-operator/main.go

```diff
@@ -101,20 +102,27 @@ func addControllersToManager(ctx context.Context, mgr manager.Manager) error {
 func isOpenshift(ctx context.Context, cfg *rest.Config) bool {
 	log := logging.FromContext(ctx)
 
-	const openShiftAPIGroupSuffix = ".openshift.io"
+	const sccGroup, sccKind = "security.openshift.io", "SecurityContextConstraints"
 
 	client, err := discovery.NewDiscoveryClientForConfig(cfg)
 	assertNoError(err)
 
-	groups, _, err := client.ServerGroupsAndResources()
+	resourceLists, err := client.ServerResources()
```

Comment:
🔧 `ServerResources()` is deprecated in favor of `ServerGroupsAndResources()`. It looks like the former calls the latter, so the behavior should be the same with either.

https://pkg.go.dev/k8s.io/client-go@v0.20.8/discovery#DiscoveryClient.ServerResources

Reply:
Darn, didn't realize that.  Will switch it back.  Thanks.

Reply:
Switched back to `ServerGroupsAndResources()`.